src/test/util_tests.cpp

```diff
@@ -1202,4 +1202,63 @@ BOOST_AUTO_TEST_CASE(test_DirIsWritable)
     fs::remove(tmpdirname);
 }
 
+BOOST_AUTO_TEST_CASE(test_ToLower)
+{
+    BOOST_CHECK(ToLower('@') == '@');
```

Comment:
Use `BOOST_CHECK_EQUAL(ToLower('@'), '@')` (it gives a clearer diagnostic message in case of failure).

Reply:
Thanks, addressed in 0234af5